src/content/en/updates/2015/07/interact-with-ble-devices-on-the-web.md

```diff
@@ -372,6 +390,7 @@ Check out our [curated Web Bluetooth Demos](https://github.com/WebBluetoothCG/de
 ## Libraries
 
 - [web-bluetooth-utils](https://www.npmjs.com/package/web-bluetooth-utils) is a npm module that adds some convenience functions to the API.
+- A Web Bluetooth API shim is available in [noble](https://github.com/sandeepmistry/noble), the most popular Node.js BLE central module. This allows you to webpack/browserify noble without the need for a WebSocket server or other plugins.
```

Comment:
With this new Web Bluetooth API shim, developers can now webpack/browserify noble - like they used to do to run it on the web - but this time without WebSocket bindings. See https://github.com/sandeepmistry/noble/pull/238/files

@monteslu will confirm for sure.